app/models/concerns/presenters/statemachine/submission.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require_dependency 'presenters/statemachine'
+module Presenters::Statemachine
```

Comment:
Nice. Again quite transparent. What is the rubocop problem. Can it be fixed?

Reply:
In this case yes, I should just write some documentation! I'll handle that.